mmc-google-analytics/src/test/unit/java/info/novatec/metricscollector/google/MetricsTest.java

```diff
@@ -0,0 +1,45 @@
+package info.novatec.metricscollector.google;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+
+/**
+ * Created by DBE on 25.08.2017.
+ */
+public class MetricsTest {
+
+    Metrics metrics;
+
+    @Before
+    public void init(){
+        metrics = new Metrics();
+    }
+
+    @Test
+    public void addDimensionsTest(){
+        Map<String, String> expectedDimensions = new HashMap<>();
```

Comment:
expectedDimensions can be moved up in class scope - it would increase tests readability.

Reply:
expectedDimensions are only used by this single test. There is no need to provide them to all tests. I understand your confusion. expectedMetrics uses keys beginning with "dimension". I fixed it